From 7ec048dbfffca7f34f0da2de57831ea3353352db Mon Sep 17 00:00:00 2001
From: Tianshu Qiu <tian.shu.qiu@intel.com>
Date: Wed, 3 Dec 2014 18:05:42 +0800
Subject: [PATCH 163/164] lm3554: Add log to indicate sucessfully flash reset

This patch adds a log to indicate that the flash
has been successfully reseted.
This log is required by CHT manufacturing tests.

Change-Id: I84521da877a6456d020fdcb1cb2e50679bbd4930
Tracked-On:https://jira01.devtools.intel.com/browse/IMINAN-11548
Signed-off-by: Tianshu Qiu <tian.shu.qiu@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/31240
Integrator: Wei Tao <wei.tao@intel.com>
Reviewed-by: Jing Tao <jing.tao@intel.com>
Approver: Wei Tao <wei.tao@intel.com>
Build: Wei Tao <wei.tao@intel.com>
Reviewed-by: Wei Tao <wei.tao@intel.com>
Maintainer: Wei Tao <wei.tao@intel.com>
---
 .../camera/drivers/media/i2c/lm3554.c              |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/lm3554.c b/drivers/external_drivers/camera/drivers/media/i2c/lm3554.c
index c587d39..04a122f 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/lm3554.c
+++ b/drivers/external_drivers/camera/drivers/media/i2c/lm3554.c
@@ -190,6 +190,7 @@ static void lm3554_hw_reset(struct i2c_client *client)
 
 	gpio_set_value(pdata->gpio_reset, 1);
 	msleep(50);
+	dev_info(&client->dev, "flash led reset successfully\n");
 }
 
 static void lm3554_flash_off_delay(long unsigned int arg)
-- 
1.7.9.5

